clio-server/config/docker-compose.yml.ctmpl

```diff
@@ -6,9 +6,9 @@
 app:
   image: broadinstitute/clio-server:{{env "DOCKER_TAG"}}
   logging:
```

Comment:
I think you need to delete `logging` entirely and nest the two options directly under `app`, based on the cromwell layout and the last error message